pkgs/by-name/pl/plakar/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "plakar";
+  version = "0.4.24-alpha";
+
+  src = fetchFromGitHub {
+    owner = "PlakarKorp";
+    repo = "plakar";
+    tag = "v${version}";
+    hash = "sha256-sAhBKIpPQL4n948ttICTUgvF7QbqqjYh+EdilEfu9XI=";
```

Comment:
This hash does not work for me:
```
error: hash mismatch in fixed-output derivation '/nix/store/wgsi3akr4v9hid5dnfvaw804cw67srfd-source.drv':
         specified: sha256-sAhBKIpPQL4n948ttICTUgvF7QbqqjYh+EdilEfu9XI=
            got:    sha256-ztWsQCQyGc07TsPulY8H2grS5gbVZaQrdwHwfTgdOQQ=
error: 1 dependencies of derivation '/nix/store/dk2x9b9gaipp4fvkddabws0jcb0z1mbh-plakar-0.4.24-alpha-go-modules.drv' failed to build
error: 2 dependencies of derivation '/nix/store/xgp17kqpwxjblvvjm9sjji83zvgrgw31-plakar-0.4.24-alpha.drv' failed to build
error: 1 dependencies of derivation '/nix/store/ysa4hm32iwibx7gznq958z438hjp6m26-review-shell.drv' failed to build
```

Reply:
you're right, that was the 0.4.2

Reply:
i've changed to 0.4.23 because there's some issues with vendored directory and it's not building.
AFAIK there will be a beta release next week, it's shouldn't be a problem